assets/src/dashboard/app/views/exploreTemplates/karma/exploreTemplates.karma.js

```diff
@@ -162,11 +163,47 @@ describe('CUJ: Creator can browse templates in grid view', () => {
       await focusOnTemplateById(lastTemplateId);
       expect(lastTemplate.contains(document.activeElement)).toBeTrue();
     });
+  });
+
+  describe('Action: See template preview from explore templates', () => {
+    beforeEach(async () => {
+      fixture.setFlags({ enableTemplatePreviews: true });
+
+      await fixture.render();
+    });
 
-    // eslint-disable-next-line jasmine/no-disabled-tests
-    xit('should trigger template preview when user clicks a card', () => {});
-    // eslint-disable-next-line jasmine/no-disabled-tests
-    xit('should trigger template preview when user presses Enter while focused on a card', () => {});
+    // it('should trigger template preview when user clicks a card', async () => {
+    //   const { templatesOrderById } = await getTemplatesState();
+
+    //   const currentCard = await getTemplateElementById(templatesOrderById[1]);
+    //   const utils = within(currentCard);
+
+    //   const activeCard = utils.getByTestId('card-action-container');
+    //   expect(activeCard).toBeTruthy();
+    //   await fixture.events.click(activeCard);
+
+    //   const viewPreviewStory = await fixture.screen.queryByTestId(
+    //     'preview-iframe'
+    //   );
+    //   expect(viewPreviewStory).toBeTruthy();
+    // });
```

Comment:
@dmmulroy help! I can't get the div to be the target for clicking. Works fine with the keyboard.

Reply:
(update for visibility! Dillon helped me solve this this AM, turns out that the click event will trigger dead center of the active element. In this case, it was clicking directly on the 'view' button which is why this test was failing with 'click' but working fine with 'enter'. )